server/src/config/config.ts

```diff
@@ -49,6 +58,20 @@ const config = {
     port: Number(DATABASE_PORT),
     ssl: DATABASE_SSL_MODE === "true",
   },
+  jira: {
+    tokenUrl: JIRA_TOKEN_URL || "https://auth.atlassian.com/oauth/token",
+    clientId: ATLASSIAN_CLIENT_ID,
+    clientSecret: ATLASSIAN_CLIENT_SECRET,
+    authorizationUrl: ATLASSIAN_AUTHORIZATION_URL || "https://auth.atlassian.com/authorize",
+    callbackUrl: CALLBACK_URL || "/jira/callback",
+    callbackRedirectUrl:
+      CALLBACK_REDIRECT_URL || NODE_ENV === "development" ? "http://localhost:3000/" : "/",
```

Comment:
Leave out the ternary here and require that `CALLBACK_REDIRECT_URL` be explicitly configured (you can leave the `/` as default, I think that's fine).

In general, ternaries are OK, but it would strive to avoid them in configuration to keep things as simple as possible as it is such a critical point in any software. Declarative configuration is easy to understand, so we want to explicitly say that "`CALLBACK_REDIRECT_URL` gets the value `asd` in environment `xyz`" instead of "depending on the value of `NODE_ENV`, `CALLBACK_REDIRECT_URL` gets value `asd` or `dsa`". Or, in other words, we will always define what environment our application is running in, so it's easier to understand its behavior if "everything depends on the environment specification" than if "something depends on the environment specification and other things depend on that something".

Reply:
tldr?